### DISCUSSION TITLE
Adding very basic query parameter example

### DISCUSSION THREAD

**Discussion by Ken-Miura** [tokio-rs/axum#300] [Q&A] [answered]
I found query parameter is used in some examples (todos and oauth). But those examples don't focus how to handle query parameter (how to use  axum::extract::Query), but their respective theme. I think providing very basic query parameter example (like form example for showing how to handle form) might be useful as introduction.

What do you think about it?

**Comment by davidpdrsn** [comment, marked as answer]
I don't understand what you mean. The example here shows how to use `Query` https://docs.rs/axum/0.2.3/axum/extract/struct.Query.html. What do you feel you're missing?

**Reply by Ken-Miura** [reply]
Thank you for answer. I just thought providing example project itself might be useful, not only document. The document provides enough information. Please close discussion.
